### ISSUE TITLE
chore(linux): failing tests due to ibus connection error?

### ISSUE THREAD

**Issue by mcdurdin** [keymanapp/keyman#9053]
See:
* https://build.palaso.org/buildConfiguration/KeymanLinux_TestPullRequests/390373?buildTab=log&linesState=585&logView=flowAware&focusLine=0

This is intermittent I guess but would be good to know what is causing it.

**Comment by ermshiperete** [comment]
Maybe fixed by #9570. I'll keep this open for a while to see if we still get failing tests.

**Comment by ermshiperete** [comment]
No, still not fixed: https://build.palaso.org/buildConfiguration/Keyman_Linux_Test_Integration/410343?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&expandBuildChangesSection=true

**Comment by mcdurdin** [comment]
😭

**Comment by ermshiperete** [comment]
Probably fixed by #9648. Closing this issue optimistically. If it's still not fixed we can re-open.
